.github/workflows/ci.yml

```diff
@@ -0,0 +1,138 @@
+name: CI-CD
+
+on:
+  push:
+    branches:
+      - main
+      - ci-test
+      - 9.x
```

Comment:
This is not the same as what in CircleCI's implementation and will not work as required (we need to allow `1.x`, `2.x` etc branches, for example).

GHA does not support regex in branch names as CircleCI does.

But it supports filter patterns: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Please update these to support those patterns

Reply:
Hi @AlexSkrypnyk 

Yes. Sorry I forgot inform you on that.
Because the if condition in job does not support regex so I made it simple as I setup in If condition of deploy job. I thought that if user use  this project they can update exactly branch name to make GHA work.

But anyway, I have updated the filter of branches for match as CircleCi as your comment now.
Could you please review it ?
Thank you.